x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -196,17 +197,23 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
                         : [primaryTimestamp],
                       include_unmapped: true,
                       runtime_mappings: runtimeMappings,
+                      ignore_unavailable: true,
                     },
                     { meta: true }
                   )
                 );
 
-                wroteWarningStatus = await hasTimestampFields({
+                const {
+                  wroteWarningStatus: wroteWarningStatusResult,
+                  skipExecution: skipExecutionResult,
+                } = await hasTimestampFields({
```

Comment:
This looks great @marshallmain ... just one small nit. `hasTimestampFields` shouldn't know anything about the rule execution, and having it return a variable called `skipExecution` seems to imply otherwise. Something like `timestampFieldFound === false` (or just `found`) might be more appropriate IMO. Especially since it's a utility function that could conceivably be used for something else outside of a rule execution.